.github/workflows/run-tests.yml

```diff
@@ -91,12 +99,16 @@ jobs:
         run: |
           docker-compose -f ${{ env.DOCKER_COMPOSE_FILE }} up -d cassandra mysql postgresql
 
-      - name: Run integration tests
-        uses: nick-fields/retry@v2
-        with:
-          timeout_minutes: 20
-          max_attempts: 2
-          command: make integration-test-coverage
+      - name: Run integration test
+        id: run-integration-test
+        timeout-minutes: 15
+        continue-on-error: true
+        run: make integration-test-coverage
+
+      - name: Retry run integration test
+        if: ${{ steps.run-integration-test.outcome == 'failure' }}
+        timeout-minutes: 15
+        run: make integration-test-coverage
```

Comment:
I'm not sure we want to retry failed tests. I brought this up the other day and was downvoted for introducing that into Buildkite [[Slack](https://temporaltechnologies.slack.com/archives/C01RN061UMR/p1699048996341939)]. We might want to have a team conversation about it.

Reply:
Removed retries.